Client.xcodeproj/project.pbxproj

```diff
@@ -294,8 +294,8 @@
 		63306D432110B3CD00F25400 /* TabManagerStore.swift in Sources */ = {isa = PBXBuildFile; fileRef = 63306D422110B3CD00F25400 /* TabManagerStore.swift */; };
 		63306D452110BAF000F25400 /* TabManagerStoreTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 63306D442110BAF000F25400 /* TabManagerStoreTests.swift */; };
 		6669B5D82113702F00CA117B /* websiteData.swift in Sources */ = {isa = PBXBuildFile; fileRef = 6669B5D72113702F00CA117B /* websiteData.swift */; };
```

Comment:
Not sure how Firefox handles unnecessary pbxproj changes, but the majority of these changes seem to be accidental Xcode changes that could be uncommitted if you try to keep a clean pbxproj.